### PR TITLE
Add font-awesome-rails

### PR DESCRIPTION
**Pull request by devin-mccurdy**
Add the font-awesome-rails gem to support https://github.com/department-of-veterans-affairs/gibct-data-service/pull/392

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

File: Gemfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails'
 gem 'jquery-ui-rails'
 
+# Use font-awesome
+gem "font-awesome-rails"
```
Comment (jholton) [inline Gemfile]: Nitpick: can you change the quotes to single-quotes?
(I did just notice there are a couple other gem entries that use double quotes; I'll clean those up)

Comment (zurbergram) [reply Gemfile]: change the double quotes to single quotes through file

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**